zulip_bots/zulip_bots/bots/flock/flock.py

```diff
@@ -19,7 +19,7 @@ def find_recipient(res: str, to: str) -> str:
             return obj['id']
 
 # Returns User's ID, if not found, returns error message.
-def get_recipient_id(to: str, config: Dict[str, str]) -> str:
+def get_recipient_id(recipient_name: str, config: Dict[str, str]) -> str:
```

Comment:
I see you reversed it here, but this seems like these two commits could be squashed together.